lint-staged.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	'packages/**/*.{js,jsx}': [
+		'prettier --write --single-quote --use-tabs --trailing-comma es5',
```

Comment:
My only open question might be for @mmcgahan since he's done this in `web-platform`.

This blob pattern will match `node_modules` directories (lerna), as well as the build outputs (i.e., `/lib`) directories. Should we be ignoring those? `meetup-web-platform` is not, so I'm curious about this.

Reply:
since those directories are git-ignored, lint-staged will never format them (they don't get staged in git), so you don't have to specifically exclude them here

Reply:
makes sense! i forgot that we were gitignoring.thanks